test/test_unit_utils.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from conjur.api.models import SslVerificationMode
+from conjur.constants import DEFAULT_CERTIFICATE_FILE
+from conjur.util import util_functions as utils
+from conjur.data_object import ConjurrcData
+
+
+class UserInputDataTest(unittest.TestCase):
```

Comment:
Good tests! simple and to the point.
You could combine them into single parametrized test, but it's as good as these, so it's only a hint.